src/datepicker/index.es

```diff
@@ -23,6 +24,7 @@ class Datepicker extends InputComponent {
       'handleClick',
       'handleClear',
     );
+    this.getCalendarTheme = memorize(getCalendarTheme.bind(null, baseCalendarTheme));
```

Comment:
lodash memorize only uses first argument as cache key. Good thing I read the documentation twice. 

https://github.com/lodash/lodash/issues/2115#issuecomment-524701409